examples/chef/esp32/main/CMakeLists.txt

```diff
@@ -121,6 +121,9 @@ idf_component_register(PRIV_INCLUDE_DIRS
                       PRIV_REQUIRES chip nvs_flash bt console mbedtls QRCode tft screen-framework spidriver
                       SRC_DIRS ${SRC_DIRS_LIST})
 
+# Forces the linker to include common/stubs.cpp
+target_link_libraries(${COMPONENT_LIB} INTERFACE "-u chef_include_stubs_impl")
```

Comment:
The Linker already links common/stubs.cpp.  However, the `emberAfExternalAttributeReadCallback` `emberAfExternalAttributeWriteCallback` aren't linked. Can this force the 2 functions to be linked?

Reply:
The common/stubs.cpp is linked by `-u` flag, so all the non-weak functions in that file will override the weak definitions We don't need to force the 2 functions to be linked.